### PR TITLE
#167749942 Integrate TravisCI with readme badge

### PR DESCRIPTION
**Pull request by alatos2**
#### What does this PR do?

This PR integrates TravisCI with readme badge

#### Description of Tasks

- Setup travis.yml file to instruct travis on what to build
- Update README.md with travis-ci badge to display build passing or failing

#### Relevant pivotal tracker story

[167749942](https://www.pivotaltracker.com/story/show/167749942)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+dist: trusty
+
+cache:
+  directories:
+    - "node_modules"
+
+notifications:
+  email: false
+  
+after_success: npm run coverage
```
Comment (ascii-dev) [inline .travis.yml]: Kindly add a new line here. Thanks

**Review by davidteejay — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (alatos2) [reply .travis.yml]: Correction well received

**Review by hustlaviola — APPROVED** [review]
Good job

**Review by Naimaoye — APPROVED** [review]

**Review by ascii-dev — APPROVED** [review]
LGTM

**Pull request merged**